<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KineticsOptimizer</class>
 <widget class="QWidget" name="KineticsOptimizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,1,0,0,0,0,0" columnstretch="0,0,0,0,0,1">
   <item row="7" column="0" colspan="3">
    <widget class="QProgressBar" name="fitProgressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" rowspan="2" colspan="4">
    <widget class="QTableView" name="parametersTable"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="wizardButton">
     <property name="text">
      <string>Parameters Wizard</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>0</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="4" rowspan="6" colspan="2">
    <widget class="QWidget" name="plotWidget" native="true"/>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="fitModelButton">
     <property name="text">
      <string>Fit Model</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-size: 24px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Kinetics Optimizer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 16px;</string>
     </property>
     <property name="text">
      <string>Fit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">font-weight: bold;
font-size: 16px;</string>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
